Formula/yaegi.rb

```diff
@@ -0,0 +1,26 @@
+class Yaegi < Formula
+  desc "Yet another elegant Go interpreter"
+  homepage "https://github.com/containous/yaegi"
+  url "https://github.com/containous/yaegi/archive/v0.0.3.tar.gz"
```

Comment:
might be better using this style:
```
  url "https://github.com/containous/yaegi.git",
    :tag      => "v0.0.3",
    :revision => "24db786bb649f292804184900d5e1818e51cd9cb"
```

Reply:
cool. is it preferred to use git and tags over tarballs now?

Reply:
Although, right now the golang formula are mixed style, but we definitely talk about the git and tags over tarballs, could not find it now. Yeah, the new formula I created are all with tag and revision, they are equivalent to tarball and sha256 though.

cc @fxcoudert

Reply:
Just a smol update regarding yaegi v0.0.4:

```
  url "https://github.com/containous/yaegi.git",
    :tag      => "v0.0.4",
    :revision => "373e3204b4979b8f7545bcde3e440e0fc3c54743"
```

Reply:
Actually tarbals are prefered cause those don't need an entire git checkout and cache better/faster